packages/manager/modules/iam/src/components/actionSelect/actionSelect.template.html

```diff
@@ -102,7 +103,7 @@
         <!-- <NoCategoryActions> -->
         <oui-checkbox
             class="ml-4"
-            data-ng-repeat="action in actionTree.actions track by action.value"
+            data-ng-repeat="action in actionTree.filteredActions track by action.value"
```

Comment:
Did you check that all is still working for embedded actions ? -> `action.embedded = true`

Reply:
Yes should be ok